Project/Fall2020_CSC403_Project/TextEngine.cs

```diff
@@ -1,37 +1,37 @@
-﻿using System;
+﻿using MyGameLibrary.Shop;
+using MyGameLibrary.Story;
+using System;
 using System.Collections.Generic;
 using System.Drawing;
 using System.IO;
 using System.Linq;
 using System.Windows.Forms;
-using MyGameLibrary.Shop;
-using MyGameLibrary.Story;
 
 namespace Fall2020_CSC403_Project
 {
     public partial class TextEngine : Form
     {
         private Story Story { get; set; }
         private Stack<Option> Options = new Stack<Option>();
-        private Stack<Option> TempOptions = new Stack<Option>();          
-        private double ForegroundImage_Xscale { get; set; }
-        private double ForegroundImage_Yscale { get; set; }
-        private double ForegroundImage_AspectRatio { get; set; }
-        private int originalHeight { get; set; }
-        private int originalWidth { get; set; }
+        private Stack<Option> TempOptions = new Stack<Option>();
+        private List<Option> options = new List<Option>();
+        public double ForegroundImage_Xscale { get; set; }
+        public double ForegroundImage_Yscale { get; set; }
+        public double ForegroundImage_AspectRatio { get; set; }
+        public int originalHeight { get; set; }
+        public int originalWidth { get; set; }
+        private bool isShop { get; set; } = false; 
+        private string line { get; set; }
```

Comment:
I'm not sure why these were moved around but it seems outside the scope of this issue

Reply:
moved these around to address some UI issues i was having when i made the options 2 columns!